app/src/main/java/com/example/dessertrelease/ui/DessertReleaseScreen.kt

```diff
@@ -67,6 +68,39 @@ fun DessertReleaseApp(
     )
 }
 
+@Composable
```

Comment:
Unless there is a reason for moving the function here, move the function back to its original location in the file so the diff is smaller and only for the changes in the function body.